src/plugin/preParsePostFormat/index.js

```diff
@@ -0,0 +1,53 @@
+import localeData from '../localeData'
+
+// Plugin template from https://day.js.org/docs/en/plugin/plugin
+export default (option, dayjsClass, dayjsFactory) => {
+  // This plugin depends on other plugins - so I will import them here
+  // equivalent to dayjsClass.extend(duration)
+  localeData(option, dayjsClass, dayjsFactory)
```

Comment:
we don't have to import other plugins like this.

Just listing the dependent plugin in the doc.